xdsl/dialects/llvm.py

```diff
@@ -1840,6 +1840,13 @@ class FPExtOp(GenericCastOp):
     name = "llvm.fpext"
 
 
+@irdl_op_definition
+class UnreachableOp(IRDLOperation):
+    name = "llvm.unreachable"
+
+    traits = traits_def(IsTerminator())
+
```

Comment:
```suggestion
    assembly_format = "attr-dict"
```
May as well add the trivial assembly format